q2_diversity/_alpha/_visualizer.py

```diff
@@ -241,7 +241,10 @@ def _reindex_with_metadata(column, columns, merged):
     merged.sort_index(axis=0, ascending=True, inplace=True)
     merged = merged.groupby(level=[column])
     counts = merged.count()
-    counts.drop(columns, axis=1, inplace=True, level=0)
+    # Removes the column name used to set the index of `merged` above
+    col_diff = set(columns) - set([column])
+    if col_diff:
+        counts.drop(col_diff, axis=1, inplace=True, level=0)
```

Comment:
The first kwarg taken by ` DataFrame.drop` is `labels` and it accepts a single label or list-like.  Passing the set directly seems to work find here, but we can make it a `pd.Index` or list if that is preferable.

Reply:
nah, this looks good to me